src/libraries/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -652,19 +650,18 @@ public static void TestingCreateInstanceFromObjectHandle(string physicalFileName
         {
             // string physicalFileName, string assemblyFile, string typeName, returnedFullNameType, expectedException
             { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "PublicClassSample", "PublicClassSample", null },
-            { "TestLoadAssembly.dll", "testloadassembly.dll", "publicclasssample", "PublicClassSample", typeof(TypeLoadException) },
+            { "TestLoadAssembly.dll", "TestLoadAssembly.dll", "publicclasssample", "PublicClassSample", typeof(TypeLoadException) },
```

Comment:
I can instead of modifying the test case, split it into two. One demonstrating FileNotFoundException, and the other demonstrating TypeLoadException?

Reply:
This is bug in CoreCLR. I have opened https://github.com/dotnet/runtime/issues/57286

The API tested here is just going to call `Assembly.LoadFrom` on the path. We should have test coverage for the file name case difference in the `Assembly.LoadFrom` API, not here.

I would recommend delete `physicalFileName` from the test data and delete the test cases that are testing the file path case differences.

Reply:
Done for both `TestingCreateInstanceFromObjectHandle` and `TestingCreateInstanceFromObjectHandleFullSignature`!